infra/cifuzz/cifuzz.py

```diff
@@ -374,10 +374,13 @@ def remove_unaffected_fuzzers(project_name, out_dir, files_changed,
     logging.error('Could not download latest coverage report.')
     return
   affected_fuzzers = []
+  logging.info('Files changed in PR:\n %s', ' '.join(files_changed))
```

Comment:
nit: do you need space after `\n` and are you sure that `' '.join()` is readable, maybe use `, ` or even `\n` for that?

Reply:
Done.